### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Andr00dz**
Andere Widerstände im Päckchen

### CODE REVIEW

File: NibblePlusPlusSMD/README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ A simple PCB to test your SMD hand-soldering skills. Going down to sand grain si
 | R3         | 0402  SMD Resistor | 1        |  62 Ohm         | yellow, white package      |
 | R4         | 0201 SMD Resistor  | 1        | 330 Ohm         | green, white package       |
 | R5         | 01005 SMD Resistor | 1        |  33 Ohm         | blue, white package        |
-| R6,R7      | 0805 SMD Resistor  | 2        | 100k Ohm        | 104                        |
+| R6,R7      | 0805 SMD Resistor  | 2        | 100k Ohm        | 104 <- im Päckchen sind andere Widerstände                   |
```
Comment (rralf) [inline NibblePlusPlusSMD/README.md]: And what kind of resistory *exactly*? How were they labeled?

Comment (Andr00dz) [reply NibblePlusPlusSMD/README.md]: they have the number 1003 in the top

Comment (rralf) [reply NibblePlusPlusSMD/README.md]: So what about ... | 104 / 1003 | ... for the patch?

**Comment by rralf** [comment]
Just cleaning up old PRs. Is this still relevant? If yes, then I'd squash those two patches.

**Comment by Andr00dz** [comment]
I don't know if this is still relevant. During 36C3, there were resistors with number 1003 instead 104 in the package. But this may have changed since then.

**Comment by marove2000** [comment]
Hi @Andr00dz for the update and sorry for the long wait. Merged!

**Pull request merged**